src/base/bittorrent/session.cpp

```diff
@@ -323,6 +325,16 @@ namespace
         return {};
     }
 #endif
+
+    using ListType = lt::entry::list_type;
+
+    ListType setToEntryList(const QSet<QString> &input)
+    {
+        ListType entryList;
```

Comment:
Please add `entryList.reserve()` if you don't mind.

Reply:
All done.